src/AttrTransform/TargetNoopener.hack

```diff
@@ -0,0 +1,40 @@
+/* Created by Chenkai Gao on 11/16/2020 */
+namespace HTMLPurifier\AttrTransform;
+use namespace HH\Lib\{C, Str, Vec};
+use namespace HTMLPurifier;
+// must be called POST validation
+
+/**
+ * Adds rel="noopener" to any links which target a different window
+ * than the current one.  This is used to prevent malicious websites
+ * from silently replacing the original window, which could be used
+ * to do phishing.
+ * This transform is controlled by %HTML.TargetNoopener.
+ */
+class HTMLPurifier_AttrTransform_TargetNoopener extends HTMLPurifier\HTMLPurifier_AttrTransform {
+	/**
+	 * @param dic<string, mixed> $attr
+	 * @param HTMLPurifier_Config $config
+	 * @param HTMLPurifier_Context $context
+	 * @return dict<string, mixed>
+	 */
+	public function transform(
+		dict<string, mixed> $attr,
```

Comment:
Can this be `dict<string, string>` or do other types end up getting set in the dict?

Reply:
This should be a string, but because of some variable name reuse, HACK thinks it can be string and bool. Should I fix it in this PR?

Reply:
Is it an easy fix?

Reply:
It is an easy fix, but will touch 9 additional files.